library/src/test/java/com/chuckerteam/chucker/internal/support/FormatUtilsTest.kt

```diff
@@ -140,4 +141,57 @@ class FormatUtilsTest {
             """.trimIndent()
         assertThat(FormatUtils.formatXml(xml)).isEqualTo(expected)
     }
+
+    @Test
+    fun testBasicCurlCommand() {
```

Comment:
There is no `BasicCurlComman` in `FormatUtils`, so this test should have more meaningful name like `testCurlCommandWIthoutHeaders` or something similar.

Reply:
agreed changed the name to `testCurlCommandWithoutHeaders`